packages/devextreme/js/__internal/core/license/rsa_bigint.ts

```diff
@@ -8,11 +8,27 @@ interface Args {
 export function compareSignatures(args: Args): boolean {
   try {
     const zero = BigInt(0);
+    const one = BigInt(1);
     const eight = BigInt(8);
 
+    const modExp = (base: bigint, exponent: bigint, modulus: bigint): bigint => {
```

Comment:
`calculateModulusOfExponentiation`? And why is it an inner function? Isn't it better to extract it into a module root?

Reply:
> And why is it an inner function? Isn't it better to extract it into a module root?

To keep all calls to `BigInt` within try-catch

Reply:
As for me it's OK to use common shortening in names, such as maths operations